webapp/src/css/variables.less

```diff
@@ -128,7 +129,7 @@
 @font-small: 0.875rem;
 @font-extra-small: 0.8125rem;
 @font-XXS: 0.75rem;
-@font-XXXS: 0.625rem;
+@font-XXXS: 0.69rem;
```

Comment:
I checked the other place using this, and it looks good. I didn't want to add another size because it's becoming confusing.